convert2rhel/unit_tests/logger_test.py

```diff
@@ -73,9 +81,36 @@ def test_tools_opts_debug(tmpdir, read_std, is_py2):
 def test_logger_custom_logger(tmpdir, caplog):
     """Test CustomLogger."""
     log_fname = "convert2rhel.log"
-    logger_module.initialize_logger(log_name=log_fname, log_dir=str(tmpdir))
+    logger_module.setup_logger_handler(log_name=log_fname, log_dir=str(tmpdir))
     logger = logging.getLogger(__name__)
     logger.task("Some task")
     logger.file("Some task write to file")
     with pytest.raises(SystemExit):
         logger.critical("Critical error")
+
+
+@pytest.mark.parametrize(
+    ("log_name", "path_exists", "is_dir"),
+    (
+        ("convert2rhel.log", True, True),
+        ("convert2rhel.log", True, True),
+        ("convert2rhel.log", False, False),
+        ("convert2rhel.log", True, False),
+    ),
+)
+def test_archive_old_logger_files(log_name, path_exists, is_dir, tmpdir, caplog):
```

Comment:
```suggestion
@pytest.mark.parametrize(
    ("log_name", "path_exists"),
    (
        ("convert2rhel.log", True),
        ("convert2rhel.log", False),
    ),
)
def test_archive_old_logger_files(log_name, path_exists, tmpdir, caplog):
```

Small nitpick, is_dir is no longer used (because we are no longer checking in the code whether archive is a directory.)

Other than that, this looks good!